src/Domain/Results.php

```diff
@@ -102,8 +102,8 @@ public function getTotalSecurityIssues(): int
                 return count($insight->getDetails());
             }
         }
-
-        throw new \RuntimeException('This should not happen');
+        // ForbiddenSecurityInsight is not included
+        return 0;
```

Comment:
Humm, what happens with the console output with this change?

Reply:
Regarding https://github.com/nunomaduro/phpinsights/blob/master/src/Application/Console/Style.php#L188

The color is green and its display 
`0 security issues encountered`.

Should we remove this message when this Insight is removed @nunomaduro ?

Reply:
Yeah, let's create an issue for this.

Reply:
Done: https://github.com/nunomaduro/phpinsights/issues/181